### ISSUE TITLE
Tests failing

### ISSUE THREAD

**Issue by smolinari** [doctrine/orientdb-odm#194]
Hi,

Starting to poke into this and I am failing tests. It seems data isn't in the database, which the tests are looking for. 

1) test\Doctrine\OrientDB\Integration\QueryBuilderTest::testSelect
Doctrine\OrientDB\Binding\InvalidQueryException: {
  "errors": [{
      "code": 500,
      "reason": 500,
      "content": "java.lang.IllegalArgumentException: Invalid class 'Profile'"
      }
  ]
}

Do I need to run something to load the data in the database first? I would thought that should be part of the test setup.

Any push for me in the right direction would be appreciated. :smile: 

Scott

**Comment by odino** [comment]
Can you check what the builds on travis do? I guess that would be the best starting point

**Comment by smolinari** [comment]
I am testing with PHPUnit over the command line directly. I don't think Travis will set up the database for testing, will it?

Scott

**Comment by smolinari** [comment]
Ok. I should have continued reading the documentation. LOL! I didn't run the load.php file, which properly sets up the database for testing.

Scott

**Comment by odino** [comment]
BTW travis sets it up :) anyhow hope youve been able to run everything correctly by now!

**Comment by smolinari** [comment]
Yes, I had a couple of strange failures mostly because "rome1" was being looked for instead of "rome".  I corrected the values to "rome" and all test passed. 

Scott
